### PR TITLE
pronouns: add 2-word pronouns aliases

### LINKED ISSUE #2068: Support 2-pronoun sets in the pronoun module
### The problem

Setting pronouns in the pronouns module can be unintuitive because the bot accepts either a 1-pronoun "object" command (`.setpronouns she`) or a long 5-pronouns "subject/object/possessive-determiner/possessive-pronoun/reflexive" commnad (`.setpronouns she/her/her/hers/herself`). However, in my experience most people are used to the 2-pronoun convention of "subject/object" `.setpronouns she/her`). 

### The solution

Supporting 2-word version of pronouns. 

### Alternatives

An alternative would be to improve the message for unrecognised pronoun sets by adding explicit instructions to use either 1- or 5-pronoun sets.

### PR DESCRIPTION
**Pull request by eliocamp**
### Description

Adds 2-word common pronoun sets. 

Closes #2068 

I didn't check it on my machine because it feels trivial. Let me know if you want me to test it properly. 

### Checklist
- [x] I have read [CONTRIBUTING.md](https://github.com/sopel-irc/sopel/blob/master/CONTRIBUTING.md)
- [x] I can and do license this contribution under the EFLv2
- [ ] No issues are reported by `make qa` (runs `make quality` and `make test`)
- [ ] I have tested the functionality of the things this change touches

### CODE REVIEW

File: sopel/modules/pronouns.py [hunk 1]
```diff
@@ -19,14 +19,21 @@
...
     'he': 'he/him/his/his/himself',
+    'he/him': 'he/him/his/his/himself',
     'xey': 'xey/xem/xyr/xyrs/xemself',
+    'xey/xem': 'xey/xem/xyr/xyrs/xemself',
     'sie': 'sie/hir/hir/hirs/hirself',
+    'sie/hir': 'sie/hir/hir/hirs/hirself',
     'it': 'it/it/its/its/itself',
+    'it/it': 'it/it/its/its/itself',
     'ey': 'ey/em/eir/eirs/eirslef',
+    'ey/em': 'ey/em/eir/eirs/eirslef'
```
Comment (Exirel) [inline sopel/modules/pronouns.py:36]: Nitpick: it's a convention to keep the last trailing comma:

```suggestion
    'ey/em': 'ey/em/eir/eirs/eirslef',
```

**Review by Exirel — CHANGES_REQUESTED** [review]
All good for me. The small nitpick is just a convention thing. For the merge commit, we'll see with @dgw how to squash that.

**Review by Exirel — APPROVED** [review]

File: sopel/modules/pronouns.py [hunk 2]
```diff
@@ -19,14 +19,21 @@
...
     'he': 'he/him/his/his/himself',
+    'he/him': 'he/him/his/his/himself',
     'xey': 'xey/xem/xyr/xyrs/xemself',
+    'xey/xem': 'xey/xem/xyr/xyrs/xemself',
     'sie': 'sie/hir/hir/hirs/hirself',
+    'sie/hir': 'sie/hir/hir/hirs/hirself',
     'it': 'it/it/its/its/itself',
+    'it/it': 'it/it/its/its/itself',
     'ey': 'ey/em/eir/eirs/eirslef',
+    'ey/em': 'ey/em/eir/eirs/eirslef',
```
Comment (dgw) [inline sopel/modules/pronouns.py:36]: L35 has been wrong for who knows how long. L36 copied the mistake.


```suggestion
    'ey': 'ey/em/eir/eirs/eirself',
    'ey/em': 'ey/em/eir/eirs/eirself',
```

**Review by dgw — COMMENTED** [review]
@Exirel I'd just rebase the branch to fixup the commit messages. However, though I haven't had a chance to look at it yet, it seems #2070 will supersede this.

**Review by dgw — APPROVED** [review]
I'll massage the commit history a bit, but it's basically ready to 🚢.

**Pull request merged**